panel/models/column.ts

```diff
@@ -43,7 +43,7 @@ export class ColumnView extends BkColumnView {
   }
 
   record_scroll_position(): void {
-    this.model.scroll_position = this.el.scrollTop;
+    this.model.scroll_position = Math.round(this.el.scrollTop);
```

Comment:
scrollTop can be a float; scroll_position is defined as param.Integer so this caused a lot of console errors.